[Sanity_Broken] The integ tests are failing on latest master codebase

### Your Environment

- **Version:** Local Development Setup
- **Affected Component:** Access Gateway
- **Affected Subcomponent:** Magma MME/SPGW service
- **Deployment Environment:** Vagrant (AGW)

### Setup Details
The S1APTester ([https://github.com/facebookexperimental/S1APTester](https://github.com/facebookexperimental/S1APTester)) was used along with Magma EPC in Vagrant based setup to reproduce the issue.

### Describe the Issue
All the LTE integ tests are failing on latest master codebase with commit id: 11954d35b80b5b0590b86598579a2666371fb92c

Attach is getting because ULA/ULR fails with ULA.result.choice.base=5012
```
000857 Thu Feb 10 08:17:12 2022 7FF1508EF700 ERROR MME-AP tasks/mme_app/mme_app_location.c:0229      ULR/ULA procedure returned non success (ULA.result.choice.base=5012)
000860 Thu Feb 10 08:17:12 2022 7FF1508EF700 DEBUG NAS-ES tasks/nas/esm/sap/esm_send.c    :0146                        ESM-SAP   - Send PDN Connectivity Reject message (pti=1, ebi=0)
000869 Thu Feb 10 08:17:12 2022 7FF1508EF700 INFO  NAS-EM tasks/nas/emm/sap/emm_send.c    :0781                                 EMMAS-SAP - Send Attach Reject message (cause=EMM_CAUSE_ESM_FAILURE)
```

The subscriberd error logs from syslog are as follows:
```
Feb 10 08:32:38 magma-dev-focal mme[25915]: [INFO] Sending S6A-AUTHENTICATION_INFORMATION_REQUEST with IMSI: 001010000000001
Feb 10 08:32:38 magma-dev-focal subscriberdb[24458]: INFO:root:Auth success: 001010000000001
Feb 10 08:32:38 magma-dev-focal mme[25915]: [INFO] Received S6A-AUTHENTICATION_INFORMATION_ANSWER for IMSI: 001010000000001; Status: ; StatusCode: 2001
Feb 10 08:32:38 magma-dev-focal sctpd[25783]: I0210 08:32:38.837173 26025 sctp_connection.cpp:195] Unhandled notification type 32777
Feb 10 08:32:38 magma-dev-focal sctpd[25783]: I0210 08:32:38.857687 26025 sctp_connection.cpp:195] Unhandled notification type 32777
Feb 10 08:32:38 magma-dev-focal mme[25915]: [DEBUG] Sending S6A-UPDATE_LOCATION_REQUEST with IMSI: 001010000000001
Feb 10 08:32:38 magma-dev-focal subscriberdb[24458]: ERROR:grpc._server:Exception calling application: Value out of range: 200000000000
Feb 10 08:32:38 magma-dev-focal subscriberdb[24458]: Traceback (most recent call last):
Feb 10 08:32:38 magma-dev-focal subscriberdb[24458]:   File "/home/vagrant/build/python/lib/python3.8/site-packages/grpc/_server.py", line 443, in _call_behavior
Feb 10 08:32:38 magma-dev-focal subscriberdb[24458]:     response_or_iterator = behavior(argument, context)
Feb 10 08:32:38 magma-dev-focal subscriberdb[24458]:   File "/home/vagrant/magma/lte/gateway/python/magma/subscriberdb/protocols/s6a_proxy_servicer.py", line 118, in UpdateLocation
Feb 10 08:32:38 magma-dev-focal subscriberdb[24458]:     ula.total_ambr.max_bandwidth_ul = profile.max_ul_bit_rate
Feb 10 08:32:38 magma-dev-focal subscriberdb[24458]: ValueError: Value out of range: 200000000000
Feb 10 08:32:38 magma-dev-focal mme[25915]: [ERROR] 2: Exception calling application: Value out of range: 200000000000
Feb 10 08:32:38 magma-dev-focal mme[25915]: [ERROR]  Received S6A-LOCATION-UPDATE_ANSWER for IMSI: 001010000000001; Status: Exception calling application: Value out of range: 200000000000; ErrorCode: 0
Feb 10 08:32:38 magma-dev-focal mme[25915]: [INFO] sent itti S6A-LOCATION-UPDATE_ANSWER for IMSI: 001010000000001
Feb 10 08:32:38 magma-dev-focal directoryd[24465]: INFO:redis_lock.acquire:Got lock for 'lock:IMSI001010000000001:directory_record:lock'.
```

The pcap and s1aptester terminal log are attached here:
[Issue.zip](https://github.com/magma/magma/files/8039474/Issue.zip)


**To Reproduce**
1) Run the sanity or any individual lte integ test case

--- a/lte/gateway/configs/subscriberdb.yml
+++ b/lte/gateway/configs/subscriberdb.yml
@@ -39,8 +39,8 @@ mme_host_address: 127.0.0.1
 mme_port: 3868
 
 # Default Subscription Profile
-default_max_ul_bit_rate: 200000000000  # 2 Gbps
-default_max_dl_bit_rate: 400000000000  # 4 Gbps
+default_max_ul_bit_rate: 2000000000  # 2 Gbps
+default_max_dl_bit_rate: 4000000000  # 4 Gbps
 
 # Relay S6A via gRPC
 relay_mode: False

